test/fixtures/gateway-account.fixtures.js

```diff
@@ -237,6 +237,84 @@ function validGatewayAccountCredentialsResponse (opts = {}) {
   return data
 }
 
+function validPatchGatewayMerchantIdRequest (opts = {}) {
+  return [
+    {
+      op: 'replace',
+      path: 'credentials/gateway_merchant_id',
+      value: opts.gatewayMerchantId || 'abcdef123abcdef'
+    },
+    {
+      op: 'replace',
+      path: 'last_updated_by_user_external_id',
+      value: opts.userExternalId || 'a-user-external-id'
+    }
+  ]
+}
+
+function validPatchGatewayMerchantIdResponse (opts = {}) {
+  const defaultCredentials = {
+    username: 'a-username',
+    merchant_id: 'a-merchant-id',
+    gateway_merchant_id: opts.gatewayMerchantId || 'abcdef123abcdef'
+  }
+  const data = {
+    external_id: opts.externalId || 'an-external-id',
+    gateway_account_id: opts.gatewayAccountId || 42,
+    gateway_account_credential_id: opts.gatewayAccountCredentialId || 888,
+    payment_provider: opts.paymentProvider || 'worldpay',
+    credentials: opts.credentials || defaultCredentials,
+    state: opts.state || 'ACTIVE',
+    created_date: opts.createdDate || '2021-06-11T13:43:51.464Z'
+  }
+  if (opts.lastUpdatedByUserExternalId !== undefined) {
+    data.last_updated_by_user_external_id = opts.lastUpdatedByUserExternalId
+  }
+  if (opts.activeStartDate !== undefined) {
+    data.active_start_date = opts.activeStartDate
+  }
+  if (opts.activeEndDate !== undefined) {
+    data.active_end_date = opts.activeEndDate
+  }
+  return data
+}
+
+function validPatchAccountGatewayAccountCredentialsStateRequest (opts = {}) {
+  return [
+    {
+      op: 'replace',
+      path: 'state',
+      value: opts.state || 'VERIFIED_WITH_LIVE_PAYMENT'
+    }
+  ]
+}
+
+function validPatchAccountGatewayAccountCredentialsStateResponse (opts = {}) {
```

Comment:
`validPatchAccountGatewayAccountCredentialsStateResponse` and `validPatchGatewayMerchantIdResponse` seem to contain exactly the same code, as this is just a fixture builder could they just be the same method?

It already has a pattern to do
```
if (opts.x !== undefined) {
  data.x = opts.s
}
```

so maybe this could be done for `gateway_merchant_id` to avoid the duplication (I think that's the only difference?)